airbyte-ci/connectors/pipelines/README.md

```diff
@@ -406,14 +406,24 @@ flowchart TD
     validate[Validate the metadata file]
     check[Check if the connector image already exists]
     build[Build the connector image for all platform variants]
+    publish_to_python_registry[Push the connector image to the python registry if enabled]
     upload_spec[Upload connector spec to the spec cache bucket]
     push[Push the connector image from DockerHub, with platform variants]
     pull[Pull the connector image from DockerHub to check SPEC can be run and the image layers are healthy]
     upload_metadata[Upload its metadata file to the metadata service bucket]
 
-    validate-->check-->build-->upload_spec-->push-->pull-->upload_metadata
+    validate-->check-->build-->upload_spec-->publish_to_python_registry-->push-->pull-->upload_metadata
 ```
 
+#### Python registry publishing
+
+If `remoteRegistries.pypi.enabled` in the connector metadata is set to `true`, the connector will be published to the python registry.
+To do so, the `PYTHON_REGISTRY_TOKEN` and `PYTHON_REGISTRY_URL` env vars are used to authenticate with the registry and publish the connector.
```

Comment:
```suggestion
To do so, the `--python-registry-token` and `--python-registry-url` options are used to authenticate with the registry and publish the connector.
```

Reply:
These are only available on the poetry publish command, which is not used for connectors. For connectors you actually have to use the env variables.

Reply:
Oh I did not notice this. I'd prefer to have the same option on the connector side 😄

Reply:
Added these options (renamed the registry option to python registry for consistency)